test-time_slot-app/src/App.js

```diff
@@ -6,28 +6,27 @@ import { postTimeslot } from './api/timeslot';
 import Login from './components/Login';
 import RenderSlots from './components/RenderSlots';
 
-let choosenSlots = [];
-
 function App() {
-  const [isAuth, setIsAuth] = React.useState(localStorage.getItem('token') !== null ? true : false);
+  const [token, setToken] = React.useState("");
+  const [userId, setUserId] = React.useState("");
+  const [isAuth, setIsAuth] = React.useState(token !== "" ? true : false);
+  const [slots, setSlots] = React.useState("");
```

Comment:
Why slots stored on this level? Handling them is not business of this component.